source/Hardpoint.cpp

```diff
@@ -190,11 +190,11 @@ void Hardpoint::Fire(Ship &ship, vector<Projectile> &projectiles, vector<Visual>
 	// Get projectiles to start at the right position. They are drawn at an
 	// offset of (.5 * velocity) and that velocity includes the velocity of the
 	// ship that fired them.
-	Point start = ship.Position() + aim.Rotate(point) - .5 * ship.Velocity();
+	Point start = ship.Position() + aim.Rotate(point * ship.Scale()) - .5 * ship.Velocity();
```

Comment:
Should we even scale hardpoint positions?

Reply:
Probably not at the moment, the current syntax suggests its a property of the sprite that it effects while it actually effects the sprite *and* other parts of the ship.

Reply:
Why wouldn't we? Scaling the ship but not the hardpoints would mean that 1) the positions given by the ship builder would be wrong and 2) in game you'd see the engine flares, weapon projectiles, and turret sprites all at the wrong spots.

Reply:
I agree with tehhowch and jafdy; scale does look like a property of the sprite.
The ship builder can be updated to also support scaling if users need that to calculate the correct positions.